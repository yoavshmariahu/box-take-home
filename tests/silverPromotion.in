k a1
K e5
s b4
S d2

[G B R P]
[p r b g]

move b4 b5 promote
move d2 d1 promote
