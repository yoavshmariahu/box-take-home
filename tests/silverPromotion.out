UPPER player action: move d2 d1 promote
5 |__|+s|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|__|+S|__|
    a  b  c  d  e

Captures UPPER: G B R P
Captures lower: p r b g

lower> 
